unit-test/background/fingerprint.es6.js

```diff
@@ -18,7 +20,8 @@ describe('User-Agent replacement', () => {
         agentStorage.family = agentSpoofer.parsedAgent.family
         agentStorage.os = agentSpoofer.parsedAgent.os.family
         agentStorage.processAgentList(agentData)
-        spy = spyOn(Tab, 'constructor')
+        tabSpy = spyOn(Tab, 'constructor')
```

Comment:
👀 How about `tabObserver` 😅

Reply:
I updated anyway - yeah these are spys to replace the original function :)